### PR TITLE
set checkbox checked value

### LINKED ISSUE #1060: "Volunteers can choose which workflow they work on" check box is broken
When you check the "Volunteers can choose which workflow they work on" box and then navigate to another aspect of Panoptes, the box resets and volunteers are not given an option of what workflow to work on when they click directly on the "Classify" tab.  Instead, users are randomly assigned a different workflow for each subject they see.  Refreshing the Project Builder edit screen, or going directly from the Project Details page to the project itself, does not resolve the issue.

### PR DESCRIPTION
**Pull request by camallen**
closes #1060

### CODE REVIEW

File: app/pages/lab/project-details.cjsx [hunk 1]
```diff
@@ -126,7 +126,8 @@ module.exports = React.createClass
 
           <p>
             <AutoSave tag="label" resource={@props.project}>
-              <input type="checkbox" name="configuration.user_chooses_workflow" value={@props.project.configuration?.user_chooses_workflow} onChange={handleInputChange.bind @props.project} />{' '}
+              {checked = @props.project.configuration?.user_chooses_workflow}
+              <input type="checkbox" name="configuration.user_chooses_workflow" checked={checked} value={checked} onChange={handleInputChange.bind @props.project} />{' '}
```
Comment (edpaget) [inline app/pages/lab/project-details.cjsx]: I think these should be defaultChecked and defaultValue right? https://facebook.github.io/react/docs/forms.html

Comment (camallen) [inline app/pages/lab/project-details.cjsx]: looks like it!

**Comment by camallen** [comment]
@edpaget updated, nice catch. I need to read the react docs.

**Comment by brian-c** [comment]
Why's this uncontrolled? `handleInputChange` updates the project.

**Comment by camallen** [comment]
Ok the docs point to it being controlled and it won't change, https://facebook.github.io/react/docs/forms.html#controlled-components

So your saying any `onChange` event that update the state will flow down via the render function? Explains the observed behaviour being the same. Do you want me to update a fix?

**Pull request merged**